commit 79556b10f5c785ef5514810928e18ddedfa4776a
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2019-04-10T22:40:34+02:00 (6 months ago)
    
    TRITON-1384 CNAPI/DAPI blows assertion "callback (Function) is required" when doing getVm

diff --git a/package.json b/package.json
index a6ef18a..9a4cd66 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "Triton Compute Node API",
-  "version": "1.24.0",
+  "version": "1.24.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -10,7 +10,7 @@
     "async": "1.5.2",
     "backoff": "2.5.0",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#f2b4e8dac18b400a64fedb27db869b8d32ef009b",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#066e52618ed6f4ef25121716b95b19cb8f90a5b4",
     "deep-object-diff": "1.1.0",
     "dox": "0.9.0",
     "gc-stats": "1.2.0",
